### PR TITLE
EDURED-158: Open gmail/outlook buttens do not open gmail/outlook

### PR DESCRIPTION
**Pull request by risinggalaxy**

### CODE REVIEW

File: EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift [hunk 1]
```diff
@@ -48,6 +48,7 @@ class CheckEmailViewController: CreateEduIDBaseViewController {
         let gmailStack = UIStackView(arrangedSubviews: [gmailImage, gmailButton])
         gmailStack.alignment = .center
         gmailStack.spacing = 8
+        gmailButton.tag = 0
```
Comment (johankool) [inline EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift:51]: Don't use magic numbers, but define constants for them instead.

**Review by johankool — CHANGES_REQUESTED** [review]

File: EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift [hunk 2]
```diff
@@ -80,8 +82,17 @@ class CheckEmailViewController: CreateEduIDBaseViewController {
...
-        posterLabel.height(34)
+        gmailButton.addTarget(self, action: #selector(openWeb(for:)), for: .touchUpInside)
+        outlookButton.addTarget(self, action: #selector(openWeb(for:)), for: .touchUpInside)
 
     }
+    
+    @objc private func openWeb(for sender: UIButton) {
+        switch sender.tag {
+        case 0: UIApplication.shared.open(URL(string: "https://gmail.com")!)
+        case 1: UIApplication.shared.open(URL(string: "https://login.live.com")!)
```
Comment (phavekes) [inline EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift:93]: This is not the url for outlook web access

**Review by phavekes — CHANGES_REQUESTED** [review]

File: EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift (hunk 2 above)
Comment (risinggalaxy) [reply EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift:93]: sure, do you have the correct URL? of course I can look it up myself, but just to be on the safe side.

File: EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift (hunk 1 above)
Comment (risinggalaxy) [reply EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift:51]: sure.

File: EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift (hunk 2 above)
Comment (risinggalaxy) [reply EduID/Flows/CreateEduID/ViewControllers/CheckEmailViewController.swift:93]: I assume `https://outlook.office.com` is the one?

**Review by dzolnai — APPROVED** [review]

**Pull request merged**